gradle/maven/defaults-maven.gradle

```diff
@@ -137,8 +254,17 @@ configure(subprojects.findAll { it.path in rootProject.published }) { prj ->
         }
       }
 
+      // Add aliases of convention tasks with shorter names.
+      task mavenToApacheSnapshots() {
```

Comment:
Maybe put this into separate file like maven-local.gradle?

Reply:
Don't know if it makes sense. The "local" maven gradle config is there mostly so that I could review the artifacts without polluting my local Maven repo. It could be removed later. The rest of maven config does make sense as a single unit. Let's leave it for now and try to publish those snapshots. If needed, we can always change later.